### ISSUE TITLE
the contribution of  Sparse Convolutional 

### ISSUE THREAD

**Issue by Super-Tree** [traveller59/second.pytorch#74]
Hi,Yan,I have a question about Sparse Convolution layers.
In your paper,the Sparse Convolutional Middle Extractor converts the feature map from [channels,11,400,352] to [channels,2,400,352] by a series of SPconv and SubMconv. Did you compared such method with plain 3d convolution as used the voxelnet? I am curious about the performance of SPconv and SubMconv. And I want to know the motivation about using  SPconv and SubMconv, in paper,I get that the spconv is runing much faster than dense,but not mentioned  the information about accuracy influence. 
Could you offer some information about the difference between SPconv and  plain 3d conv about accuracy influence?

**Comment by traveller59** [comment]
The dense conv3d network is 4x slower than the sparse middle network when training, which means I need 80 hours (160 epochs) to train the dense network. This is unacceptable since I only have one GPU.
